src/main/java/duke/action/Action.java

```diff
@@ -0,0 +1,82 @@
+package duke.action;
+
+/**
+ * Base class of all the task classes.
+ * String variable act describes the task.
+ * boolean variable isDone describes the status of the task.
+ */
+public class Action {
+
+    protected final String act;
+    protected final boolean isDone;
+
+    /**
+     * Constructs the Action class by taking in a
+     * String variable, string. Sets the isDone variable
+     * to false by default.
+     * @param string task
+     */
+    public Action(String string) {
+         act = string;
+         isDone = false;
+    }
+
+    /**
+     * Constructs the Action class by taking in a
+     * String variable, string and a boolean
+     * variable, status.
+     * @param string task
+     * @param status done status of task
+     */
+    public Action(String string, boolean status)  {
```

Comment:
```suggestion
    public Action(String string, boolean status) {
```
Coding standard spacing